packages/file-asset-apis/src/s3/createS3Client.ts

```diff
@@ -82,20 +85,32 @@ export async function createHttpOptions(
     // https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/node-registering-certs.html
     // Instead of updating the client, we can just update the config before creating the client
 
-    const certPath = config.certLocation ?? '/etc/ssl/certs/ca-certificates.crt';
-    const pathFound = await fs.exists(certPath);
+    const allCerts: string[] = [];
+    const defaultCerts = tls.rootCertificates;
 
-    if (!pathFound) {
-        throw new Error(
-            `No cert path was found in config.certLocation: "${config.certLocation}" or in default "/etc/ssl/certs/ca-certificates.crt" location`
-        );
+    // Deprecated
+    if (config.certLocation) {
+        const certPathFound = await fs.existsSync(config.certLocation);
+        if (certPathFound) {
+            allCerts.push(await fs.readFileSync(config.certLocation, 'ascii'));
+        } else {
+            throw new Error(`No cert path was found in config.certLocation: "${config.certLocation}"`);
+        }
+    }
+
+    if (config.caCertificate) {
+        allCerts.push(config.caCertificate);
     }
-    // Assumes all certs needed are in a single bundle
-    const certs = await fs.readFile(certPath);
+
+    if (config.globalCaCertificate) {
+        allCerts.push(config.globalCaCertificate);
+    }
+
+    allCerts.push(...defaultCerts);
```

Comment:
This spread operator is a little confusing combined with `push`, but I guess the only other option is `concat`  lower down on line 113.  Probably not worth it.  I was hoping to find something like `.append` but I guess not.

Reply:
I could do this `allCerts = allCerts.concat(defaultCerts);`. I would need to use let instead of const when declaring `allCerts`.

Reply:
How about this? More explanatory
```
    const terafoundationCerts: string[] = [];
    const defaultNodeCerts = tls.rootCertificates;

    // Deprecated
    if (config.certLocation) {
        const certPathFound = await fs.existsSync(config.certLocation);
        if (certPathFound) {
            terafoundationCerts.push(await fs.readFileSync(config.certLocation, 'ascii'));
        } else {
            throw new Error(`No cert path was found in config.certLocation: "${config.certLocation}"`);
        }
    }

    if (config.caCertificate) {
        terafoundationCerts.push(config.caCertificate);
    }

    if (config.globalCaCertificate) {
        terafoundationCerts.push(config.globalCaCertificate);
    }

    const allCerts: string[] = terafoundationCerts.concat(defaultNodeCerts);

    return {
        rejectUnauthorized: true,
        ca: allCerts
    };
```